src/App.js

```diff
@@ -57,11 +57,11 @@ class App extends Component {
     );
   }
 
-  handleAuthenticationInfoUpdate = function(authResponse) {
-    if(!authResponse || authResponse.error) {
+  handleAuthenticationInfoUpdate = function (authResponse) {
+    if (!authResponse || authResponse.error) {
       console.log("Encountered an error on login: " + util.inspect(authResponse, { showHidden: true, depth: null }));
     } else {
-      this.setState({authenticationInfo: {access_token: authResponse.access_token, expires_in: authResponse.expires_in}})
+      this.setState({ authenticationInfo: { access_token: authResponse.access_token, expires_in: authResponse.expires_in } })
```

Comment:
Why is this inserting additional spaces all over? Even between an if and an open brace? Doesn't look right. Will make all future CRs noisy and in any case, shouldn't happen given the changes you are proposing in this CR. 

Double check.

Reply:
Space between if and open brace is a common format. I have removed spaces after the braces.